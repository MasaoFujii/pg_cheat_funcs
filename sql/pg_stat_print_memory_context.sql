CREATE EXTENSION pg_cheat_funcs;

\pset null '(null)'

SELECT pg_stat_print_memory_context();

DROP EXTENSION pg_cheat_funcs;
