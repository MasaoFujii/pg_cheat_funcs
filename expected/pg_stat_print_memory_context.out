CREATE EXTENSION pg_cheat_funcs;
\pset null '(null)'
SELECT pg_stat_print_memory_context();
 pg_stat_print_memory_context 
------------------------------
 
(1 row)

DROP EXTENSION pg_cheat_funcs;
